definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -5579,11 +5579,11 @@
     "CRUD": "R"
   },
   {
-    "LiveFrom": "25/09/2019",
+    "LiveFrom": "23/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solApplyForDA",
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
+    "UserRole": "[PETSOLICITOR]",
```

Comment:
What's this user role, @Dizrupto ?

Reply:
It is the specific role for petitioner solicitor. 
This is how we differentiate between a petitioner solicitor and a respondent solicitor :).
So that means this will only show for a petitioner solicitor, but not a respondent solicitor
You can see those defined in 'CaseRoles.json'

Reply:
It's a `case role` which means it's granted to a specific user and creates an unique link between this user and this particular case.